health/demo/demo.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/zenoss/zenoss-go-sdk/health"
+	"github.com/zenoss/zenoss-go-sdk/health/log"
+	"github.com/zenoss/zenoss-go-sdk/health/target"
+	"github.com/zenoss/zenoss-go-sdk/health/writer"
+)
+
+const (
+	mercedesTarget = "mercedes.citaro"
+
+	speedMetricID       = "speed"
+	stationsCounterID   = "stations"
+	passengersCounterID = "passengers"
+)
+
+func main() {
```

Comment:
There is an example folder which may be a better location for this demo ( as this is where we currently have examples of how to use functionality provided by the sdk)

Reply:
Fixed